Spring-Boot/Spring-Boot-Student/src/test/java/tw/waterball/judgegirl/springboot/student/controllers/it/StudentControllerTest.java

```diff
@@ -123,36 +123,49 @@ private StudentView signUpAndGetResponseBody(Student student) throws Exception {
         return getBody(signUp(student).andExpect(status().isOk()), StudentView.class);
     }
 
+    private ResultActions signUp(String name, String email, String password) throws Exception {
+        Student newStudent = new Student(name, email, password);
+        return signUp(newStudent);
+    }
+
     private ResultActions signUp(Student student) throws Exception {
         return mockMvc.perform(post("/api/students/signUp")
                 .contentType(MediaType.APPLICATION_JSON)
                 .content(toJson(student)));
     }
 
-    private ResultActions signUp(String name, String email, String password) throws Exception {
-        Student newStudent = new Student(name, email, password);
-        return mockMvc.perform(post("/api/students/signUp")
+    private StudentView signUpAdminAndGetResponseBody(Student admin) throws Exception {
+        return getBody(signUpAdmin(admin).andExpect(status().isOk()), StudentView.class);
+    }
+
+    private ResultActions signUpAdmin(String name, String email, String password) throws Exception {
+        Student newAdmin = new Admin(name, email, password);
+        return signUpAdmin(newAdmin);
+    }
+
+    private ResultActions signUpAdmin(Student admin) throws Exception {
+        return mockMvc.perform(post("/api/admins")
                 .contentType(MediaType.APPLICATION_JSON)
-                .content(toJson(newStudent)));
+                .content(toJson(admin)));
```

Comment:
move **all** the called methods to the bottom.

callers should be placed above the callees